src/ts_catalog/metadata.h

```diff
@@ -9,10 +9,26 @@
 #include <postgres.h>
 #include "export.h"
 
+#define METADATA_UUID_KEY_NAME "uuid"
+#define METADATA_EXPORTED_UUID_KEY_NAME "exported_uuid"
+#define METADATA_TIMESTAMP_KEY_NAME "install_timestamp"
```

Comment:
You should remove the double definition from `src/telemetry/telemetry_metadata.c`